third_party/rules_pycross/pycross/private/tools/wheel_installer.py

```diff
@@ -97,6 +98,24 @@ def main(args: Any) -> None:
 
     setup_namespace_pkg_compatibility(lib_dir)
```

Comment:
I would have expected patching to happen before this line.

Reply:
Hmm. In case the user wants to set up their own `__init__.py` file or something? My thought was to let the user patch as much of the internal code gen as possible for the most flexibility.